pkg/neotest/doc.go

```diff
@@ -4,12 +4,12 @@ It can be used to implement unit-tests for contracts in Go using regular Go
 conventions.
 
 Usually it's used like this:
- * an instance of the blockchain is created using chain subpackage
- * the target contract is compiled using one of Compile* functions
- * and Executor is created for the blockchain
- * it's used to deploy a contract with DeployContract
- * CommitteeInvoker and/or ValidatorInvoker are then created to perform test invocations
- * if needed, NewAccount is used to create an appropriate number of accounts for the test
+  - an instance of the blockchain is created using chain subpackage
```

Comment:
`*` is fine for the list too.

Reply:
Accoarding to the documentation it is, but linter (even updated to v3) doesn't like it. And `gofmt -w -s -l` replaces `*` with `-`, not sure, why. Maybe we can keep `-`?

Reply:
I'm OK with `-`, it's just that when I'm writing anything like that I'm using `*` without thinking much and it's documented to work fine, but if tools prefer dashes instead of asterisks and even do automatic replacing --- whatever, let it be dashes.